src/Kunstmaan/NodeBundle/Repository/NodeTranslationRepository.php

```diff
@@ -255,7 +278,8 @@ private function getNodeTranslationForSlugPart(
                 'WITH',
                 't.publicNodeVersion = v.id'
             )
-            ->where('n.deleted != 1')
+            ->where('n.deleted != :deleted')
+            ->setParameter('deleted', true, \PDO::PARAM_BOOL)
```

Comment:
See comment/question above about the 3th parameter

Reply:
parameter has beeen removed